mps_youtube/g.py

```diff
@@ -51,6 +51,16 @@
 HISTFILE = os.path.join(paths.get_config_dir(), "play_history")
 CACHEFILE = os.path.join(paths.get_config_dir(), "cache_py_" + sys.version[0:5])
 READLINE_FILE = None
+categories = {
```

Comment:
It looks like the categories can be retrieved through the API: https://developers.google.com/youtube/v3/docs/videoCategories/list

It might not be great for this purpose though, since the titles are several words.